docs/usage/initialization.rst

```diff
@@ -42,6 +42,41 @@ hosted on private urls such as *customEdmxUrl.com* and *customEdmUrl.com*:
 
     northwind = pyodata.Client(SERVICE_URL, requests.Session(), namespaces=namespaces)
 
+Config
+------
+
+Class config provides easy to use wrapper for all parser configuration. These are:
+    - XML namespaces
+    - Parser policies(how parser act in case of invalid XML tag). We now support three types of policies:
+        - Policy fatal - parser raises exception and terminates
+        - Policy warning - parser reports given problem, then execute fallback code(e. g. replace unknown type with NullType) and then continues normally.
+        - Policy Silent - Same as warning except that error message is thrown away.
+
+Parser policies can be specified individually for each XML tag(See enum ParserError for more details). If no policy is specified for the tag default policy is used.
```

Comment:
What is the tag policy? I guess there is a missing comma between tag and default.